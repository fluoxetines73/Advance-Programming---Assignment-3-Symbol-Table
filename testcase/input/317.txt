PRINT
BEGIN
INSERT b string
PRINT
INSERT c number
ASSIGN c 456
BEGIN
INSERT f_FF number
RPRINT
LOOKUP f_FF
PRINT
LOOKUP b
BEGIN
END
BEGIN
PRINT
RPRINT
PRINT
LOOKUP f_FF
ASSIGN b 987654
INSERT f_FF string
BEGIN
INSERT 21_3c 123
END
END
END
END