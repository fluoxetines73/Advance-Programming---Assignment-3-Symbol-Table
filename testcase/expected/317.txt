TypeMismatch: ASSIGN b 987654